tests/integration/transformers/test_numerical.py

```diff
@@ -135,3 +136,21 @@ def test_int_nan(self):
 
         reverse = ct.reverse_transform(transformed)
         np.testing.assert_array_almost_equal(reverse, data, decimal=2)
+
+
+class TestBayesGMMTransformer:
+
+    def test_simple(self):
```

Comment:
Can we rename this to be more explicit about what it's testing? Also can we add some tests with nulls similar to the others in this file?